src/decorators/styler.js

```diff
@@ -0,0 +1,120 @@
+import { StyleSheet } from "aphrodite-jss";
```

Comment:
I'm concerned about using aphrodite-jss, maybe could be replace it by own implementation, could be a possibility?

Reply:
The inline-styling approach cannot be utilised because **React** (more specifically, **HTML**) does not support inline pseudo-selectors. If you have got any suggestions, I would be happy to give them a try.

Reply:
I guess, we could have it for now. Until find other solution. Thanks for contribution.